src/components/search/index.tsx

```diff
@@ -223,12 +222,12 @@ const Search: FunctionComponent<SearchProps> = ({
                     <RefinementList isShown={isFilterShown} attribute="type" />
                   </div>
                   {isRefinementOn && (
-                    <div
+                    <button
```

Comment:
a11y linter really hates onclick when it's not a button and demands all sorts of changes/additions so I just made it a button